CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Fix authentication view glitch at startup
 - Remove "No priority" from fee options
 - Display skeleton screens (placeholder) in the account page during data loading to enhance UX
+- Fix Moonpay widget loading issues
```

Comment:
nit: I'd move put the Moonpay change before the skeleton things, as it was released before

Reply:
🤔 yeah it broke twice 😅  and the release-4.46 has this item in the changelog. https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/release-v4.46.0/CHANGELOG.md

this  fixed the 2nd time that it broken (Android) in 4.46, 4.45 is still broken on Andorid https://github.com/BitBoxSwiss/bitbox-wallet-app/commit/6392ff9eb32c121eedcac8f695a63db82e279a79

or am I missing smth?

Reply:
I prefer to not move it as it is an unordered list  😇